testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1281,3 +1282,14 @@ def server_secret
   puts "URL: #{url}"
   step %(I enter "#{url}" as "calendar-data-text")
 end
+
+And(/^I upload ([^\"]*) to ([^\"]*) on ([^\"]*)$/) do |file, destination, vm|
+  login = 'root'
+  password = 'linux'
+  target = get_target(vm)
+  target.run("mkdir -p #{destination}")
+  Net::SCP.start(target.ip, login, :password => password) do |scp|
```

Comment:
Don't reinvent the wheel, we have everything we need to copy a file. See existing steps which use `upload_files`.